public/app/features/transformers/docs/content.ts

```diff
@@ -628,6 +628,68 @@ Use this transformation to construct a matrix by specifying fields from your que
   `;
     },
   },
+  groupToNestedTable: {
+    name: 'Group to nested table',
+    getHelperDocs: function (imageRenderType: ImageRenderType = ImageRenderType.ShortcodeFigure) {
+      return `
+  Use this transformation to group the data by a specified field (column) value and process calculations on each group. This generates records that share the same grouped field value to be displayed in a nested table.
```

Comment:
Just changing this to passive voice because it's actually clearer.

```suggestion
  Use this transformation to group the data by a specified field (column) value and process calculations on each group. Records are generated that share the same grouped field value, to be displayed in a nested table.
```